recipes/conversation_view/emoji_row/emoji_row.vue

```diff
@@ -100,5 +94,32 @@ export default {
 </script>
 
 <style lang="less">
+.emoji-row__reaction {
+  height: var(--dt-size-550);
```

Comment:
Rather than provide a fixed height on the Emoji reaction item, use `padding` to dictate the size. 

Referring to the [official Figma component](https://www.figma.com/file/oTY56LgQha8ljb4c8w4GtO/DT-Shared%3A-DP-App-Components?type=design&node-id=9240-7140&mode=design&t=W1O4c7ecvvFi4Bca-11) that would be...
```
padding: var(--dt-space-300) var(--dt-space-400); // 4px 8px
gap: var(--dt-space-300);